app/assets/javascripts/tooltip.js

```diff
@@ -0,0 +1,19 @@
+var showTooltips = function(options) {
```

Comment:
That is true, I was referring to the fact that you should not define it in that way.

You are defining a global variable here. Bound to `window` I can easily break all of the functionality. Which I do not get, since you are creating a `jQuery` based implementation afterwards.